config/config_test.go

```diff
@@ -729,13 +718,12 @@ func TestPrometheusValidateAndLog(t *testing.T) {
 			},
 		},
 		{
-			description: "[4] Port valid, Namespace valid, Subsystem valid. Expect elements in log",
+			description: "Port valid, Namespace valid, Subsystem valid. Expect elements in log",
```

Comment:
missing a case with valid port and neither of namespace or subsystem set, which should not result in an error

Reply:
Added extra test case.